package.json

```diff
@@ -36,6 +36,7 @@
     "js-yaml": "~3.6.1",
     "marked": "~0.3.5",
     "marked-terminal": "~1.6.1",
+    "mkdirp": "^0.5.1",
```

Comment:
Cheers, will do. I tried to be a good citizen, but I wanted to produce something quickly to see if it could fly 😊